ui/dashboards/src/components/Dashboard/Dashboard.stories.tsx

```diff
@@ -113,17 +150,27 @@ const DEFAULT_ALL_DASHBOARD: DashboardResource = {
             {
               x: 0,
               y: 0,
-              width: 10,
+              width: 12,
```

Comment:
ooc, why did this one get wider?

Reply:
Oh sorry missed this comment! This probably happened at some point during rebasing since this branch started before the queries breaking changes. I'll update the other panel to be same width since it's weird that one is 10 and one is 12 now